admin_guide/cluster_management.rst

```diff
@@ -406,6 +406,30 @@ Finally, by joining ``citus_stat_statements`` with ``pg_stat_statements`` we can
   │ router        │   1 │  0.392590 │
   └───────────────┴─────┴───────────┘
 
+Resource Conservation
+=====================
+
+Limiting Long-Running Queries
+-----------------------------
+
+Long running queries can hold locks, queue up WAL, or just consume a lot of system resources, so in a production environment it's good to prevent them from running too long. You can set the `statement_timeout <https://www.postgresql.org/docs/10/static/runtime-config-client.html#GUC-STATEMENT-TIMEOUT>`_ parameter on the coordinator and workers to cancel queries that run too long.
+
+.. code-block:: postgres
+
+   -- limit queries to five minutes
+   ALTER DATABASE citus
```

Comment:
Don't we cancel the worker queries if we set this on the coordinator? Did you observe any different behavior?

Reply:
No, I observed the queries on the workers to keep running (with citus 7.5.0 on Cloud at least)